docs/about/index.rst

```diff
@@ -28,6 +28,25 @@ Governance
 ----------
 
 
+Citing Fairlearn
+----------------
+
+If you wish to cite Fairlearn in your work, please use the following:
+
+.. code ::
+
+    @techreport{bird2020fairlearn,
+        author = {Bird, Sarah and Dud{\'i}k, Miro and Edgar, Richard and Horn, Brandon and Lutz, Roman and Milan, Vanessa and Sameki, Mehrnoosh and Wallach, Hanna and Walker, Kathleen},
+        title = {Fairlearn: A toolkit for assessing and improving fairness in {AI}},
+        institution = {Microsoft},
+        year = {2020},
+        month = {May},
+        url = "https://www.microsoft.com/en-us/research/publication/fairlearn-a-toolkit-for-assessing-and-improving-fairness-in-ai/",
+        number = {MSR-TR-2020-32},
+    }
+
+As we shift to a more open governance model, we expect to update this citation.
```

Comment:
I'm not sure I understand what this means. In my mind, it seems reasonable to write a follow-up at some point (I have no timeline in mind) when we expand the focus beyond binary classification & regression to more complex tasks, or perhaps when we manage to get a few examples worked out. That said, I am not aware of plans and this seems to be hinting at something, so I just want to clarify :-)

Reply:
Agreed with @romanlutz. we will add a new citation if there's a new paper! and we should write a new paper when there's a good reason to write a new paper--so when we go neutral, we should reflect if there's enough for a new paper.

For now, let's just drop "As we shift ..."

Reply:
I had it in to minimise the appearance of excessive MS influence. I'll remove.